src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -64,5 +91,33 @@ private static Process GetProcessById(int processId)
                 return null;
             }
         }
+
+        private static string GetArgs(Process process)
+        {
+            if(RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                using (ManagementObjectSearcher searcher = new ManagementObjectSearcher($"SELECT CommandLine from Win32_Process WHERE ProcessId = {process.Id}"))
```

Comment:
Non admin processes should not be an issue. Processes running as a admin user, may be an issue. I will add a try catch and display an appropriate message.

Reply:
If those show up as `IOException`s, then I think it will already be covered by the catch block, but I would double check the docs for what the behavior is.  It's definitely worth doing some local validation before merging.

Reply:
Looking at the documents, there are multiple ways of setting up the /proc file system. By default, all users should have access to other user's processe's cmdline and other files.
http://man7.org/linux/man-pages/man5/proc.5.html

If the setup is different, the user will not have access:
>1   Users may not access files and subdirectories inside any
                  /proc/[pid] directories but their own (the /proc/[pid]
                  directories themselves remain visible).  Sensitive files
                  such as /proc/[pid]/cmdline and /proc/[pid]/status are now
                  protected against other users.
This should throw a IO exception if I am reading right. Please correct me if I am wrong.